packages/@adobe/spectrum-css-temp/components/button/index.css

```diff
@@ -372,6 +372,18 @@ a.spectrum-ActionButton {
     margin-block: 0;
     margin-inline: auto;
   }
+
+  &.spectrum-ClearButton--inset {
+    &:focus-visible {
+      &:after {
+        left: 4px;
+        right: 4px;
+        bottom: 4px;
+        top: 4px;
+        box-shadow: inset 0 0 0 var(--spectrum-focus-ring-size) var(--spectrum-focus-ring-color);
+      }
+    }
```

Comment:
nit: should the clear button focus ring animate "expanding" when focus moves away?

https://github.com/user-attachments/assets/97118c81-6249-40ed-a6c5-8ea2284c69c3

Reply:
haha, i thought i was seeing things, yours is much more defined than mine